README.md

```diff
@@ -27,7 +27,7 @@ class SomeStore {
 }
 
 const hydrate = create({
-    storage: localForage,   // or AsyncStorage in react-native.
+    storage: localStorage,   // or AsyncStorage in react-native.
```

Comment:
This is not a typo :)
See https://github.com/pinqy520/mobx-persist#createconfig